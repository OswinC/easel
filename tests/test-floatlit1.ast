float pi = 3.14159;
float a = 300000.;
float b = 3e-05;
float c = -300000.;
float d = 3e-07;
float e = 30000.;
float f = 330000.;
float g = 3.3e-05;
float h = 3.3e-15;
