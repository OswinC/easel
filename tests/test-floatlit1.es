float pi = 3.14159;
float a = 3e5;
float b = 3.e-5;
float c = -3.e5;
float d = .3e-6;
float e = .3e5;
float f = 3.3e5;
float g = 3.3e-5;
float h = 3.3e-15;
