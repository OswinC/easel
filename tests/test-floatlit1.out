3.141590
300000.000000
0.000030
-300000.000000
0.000000
30000.000000
330000.000000
0.000033
0.000000 
